index.bs

```diff
@@ -2210,6 +2221,10 @@ The following abstract operations operate on {{ReadableStream}} instances at a h
     ask to shutdown, optionally with an error |error|, then:
    1. If |shuttingDown| is true, abort these substeps.
    1. Set |shuttingDown| to true.
+   1. If |reader| [=implements=] {{ReadableStreamBYOBReader}}, perform
```

Comment:
Reader will always be a default reader at this point right?

Reply:
Not here, this is the *start* of the shutdown process.

But indeed, at the *end* of the shutdown process (i.e. "finalize"), *reader* will always be a default reader, since we would have gone through either "shutdown" or "shutdown with an action".

Reply:
Hmm, it would conflict with this note in "shutdown with an action" though:
> This exchange of readers is not observable to author code and the user agent is free to implement this differently, for example by keeping the same reader and internally aborting its pending read requests. 

🤔

Reply:
Eh, I think it's clear enough that if you're doing something different from the spec you need to tweak more than just one step.